android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -518,83 +542,122 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?)
     handlerFactories.firstOrNull { it.type == handler.javaClass } as HandlerFactory<T>?
 
   private fun <T : GestureHandler<T>> onHandlerUpdate(handler: T, motionEvent: MotionEvent) {
+    // onUpdate
```

Comment:
what does this comment concern?

Reply:
Events sent by `onHandlerUpdate` are handled by the `onUpdate` callback of a gesture.

Reply:
So maybe it would be good to extend this comment a bit so it is more descriptive?

Reply:
Good point, done in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/e51489fd2dc62a5e88f94fe0017bfec99081fc8a.